packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -9,8 +9,11 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+
+jest.mock('@react-aria/live-announcer');
```

Comment:
Jest hoists `jest.mock` calls to the top of the module and before any imports. So, it should be okay to push this line down after the imports block, and Jest will still call it first (unless that's the coding style we're following in our tests).